README.md

```diff
@@ -0,0 +1,9 @@
+# ArduPilot light builder
```

Comment:
Please change to "ArduPilot master_custom_light branch rebuilder"

Reply:
```suggestion
# ArduPilot master_custom_light branch rebuilder
```